pisek/jobs/parts/tools.py

```diff
@@ -38,6 +38,7 @@ def _get_jobs(self) -> list[Job]:
         jobs: list[Job] = [
             PrepareMinibox(self._env),
             PrepareTextPreprocessor(self._env),
+            PrepareJudgeToken(self._env),
```

Comment:
Do we really want to always compile this?

Reply:
Fixed. The judge is now compiled on demand.